pokemongo_bot/cell_workers/incubate_eggs_worker.py

```diff
@@ -0,0 +1,67 @@
+from utils import distance, format_dist
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot import logger
+
+
+class IncubateEggsWorker(object):
+    def __init__(self, bot):
+        self.api = bot.api
+        self.config = bot.config
+        self.bot = bot
+        # self.position = bot.position
+
+    def work(self):
+        if not self.config.hatch_eggs:
+            return
+
+        self.api.get_inventory()
```

Comment:
Well seems like you're right. I looked through the code but I didn't think to look at the workers...